itunes-connect.csv

```diff
@@ -0,0 +1,27 @@
+Name,Aware
+Privacy Policy URL,
+Bundle ID,com.github.josh.Aware
+SKU,Aware1
+Primary Language,English
+Primary Category,Productivity
+Secondary Category,
+Price,USD 0.99 (Tier 1)
```

Comment:
So what we are feeling on the price. Some of the basic tiers:

1. USD 0 (Free)
2. USD 0.99 (Tier 1)
3. USD 1.99 (Tier 2)
4. USD 2.99 (Tier 3)